recipes/swashes/bld.bat

```diff
@@ -0,0 +1,4 @@
+cmake.exe CMakeLists.txt
+cmake.exe --build . --config Release
+if not exist "%PREFIX%\" mkdir "%PREFIX%"
+COPY bin\swashes.exe %PREFIX%\
```

Comment:
This file should have exit conditions after each line, see https://conda-forge.org/docs/maintainer/knowledge_base.html#using-cmake

Reply:
Done in 67c93f9